### ISSUE TITLE
Results on Pre-Trained models poor

### ISSUE THREAD

**Issue by archiki** [SeanNaren/deepspeech.pytorch#453]
Hello @SeanNaren ,

I've downloaded DeepSpeech code of version 1.2 and the pre-trained librispeech models V2.
I tried to evaluate the pre-trained model on test_clean set of Librispeech. ( csv file and wav files are processed by librispeech.py which is under the data/ folder. )
test command is 
```python test.py --model-path models/librispeech_pretrained.pth --test-manifest data/libri_test_clean_manifest.csv```
The results I'm getting are quite bad:
Test Summary 	Average WER 35.678	Average CER 13.472	
I modified the following lines of code:

1.  load_audio function in *data_loader.py*
line 24  ```sound = sound.numpy() to sound = sound.numpy().T```
line 165 ```with open ``` to ```with io.open```

 2. In *decoder.py*
line169 : ```char = self.int_to_char[sequence[i]]``` to ```char = self.int_to_char[sequence[i].item()]```
line 172: ```if remove_repetitions and i != 0 and char == self.int_to_char[sequence[i - 1]]:``` to ```if remove_repetitions and i != 0 and char == self.int_to_char[sequence[i - 1].item()]:```
3. In *model.py*
line 137 ```nn.Conv2d(32, 32, kernel_size=(21, 11), stride=(2, 1))``` to ```nn.Conv2d(32, 32, kernel_size=(21, 11), stride=(2, 1),padding=(0, 5))```

Can anyone tell me where I am going wrong, any help is appreciated!
Thanks

**Comment by smallflyingpig** [comment]
> Hello @SeanNaren ,
> 
> I've downloaded DeepSpeech code of version 1.2 and the pre-trained librispeech models V2.
> I tried to evaluate the pre-trained model on test_clean set of Librispeech. ( csv file and wav files are processed by librispeech.py which is under the data/ folder. )
> test command is
> `python test.py --model-path models/librispeech_pretrained.pth --test-manifest data/libri_test_clean_manifest.csv`
> The results I'm getting are quite bad:
> Test Summary Average WER 35.678	Average CER 13.472
> I modified the following lines of code:
> 
> 1. load_audio function in _data_loader.py_
>    line 24  `sound = sound.numpy() to sound = sound.numpy().T`
>    line 165 `with open ` to `with io.open`
> 2. In _decoder.py_
>    line169 : `char = self.int_to_char[sequence[i]]` to `char = self.int_to_char[sequence[i].item()]`
>    line 172: `if remove_repetitions and i != 0 and char == self.int_to_char[sequence[i - 1]]:` to `if remove_repetitions and i != 0 and char == self.int_to_char[sequence[i - 1].item()]:`
> 3. In _model.py_
>    line 137 `nn.Conv2d(32, 32, kernel_size=(21, 11), stride=(2, 1))` to `nn.Conv2d(32, 32, kernel_size=(21, 11), stride=(2, 1),padding=(0, 5))`
> 
> Can anyone tell me where I am going wrong, any help is appreciated!
> Thanks
I got the same result, so what is the problem?

**Comment by SeanNaren** [comment]
I would highly suggest using the latest pre-trained models and latest release!
